### PR TITLE
Few quest lines

### PR DESCRIPTION
**Pull request by JayShee**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9683,23 +9683,23 @@ QUEST_LV_0100_20151016_009682	{memo X}
...
+QUEST_LV_0100_20151016_009688	It wasn't a strength of a human. I was trembling.{nl}She disappeared down there soon after throwing away the shredded leather.{nl}
+QUEST_LV_0100_20151016_009689	There might still be a bit of leather left if you head down there. I.. I'm too afraid to go even if it was an order from a goddess.
 QUEST_LV_0100_20151016_009690	{memo X}
-QUEST_LV_0100_20151016_009691	
-QUEST_LV_0100_20151016_009692	
-QUEST_LV_0100_20151016_009693	
-QUEST_LV_0100_20151016_009694	
-QUEST_LV_0100_20151016_009695	
+QUEST_LV_0100_20151016_009691	I have no idea what's hidden in the leather.{nl}I hope there's a bit of leather left.
+QUEST_LV_0100_20151016_009692	What if the monsters have ate it?
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9692]: have eaten it?

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9683,23 +9683,23 @@ QUEST_LV_0100_20151016_009682	{memo X}
...
+QUEST_LV_0100_20151016_009693	Let me take a look.{nl}Hm..{nl}
+QUEST_LV_0100_20151016_009694	I have seen this crest before..{nl}Ah.. Ah!{nl}
+QUEST_LV_0100_20151016_009695	I remembered. I saw it from a book.{nl}It's the crest of Lunatic Demon King, Vaiga!{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9695]: It's the crest of the Lunatic Demon King, Vaiga!{nl}

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -9683,23 +9683,23 @@ QUEST_LV_0100_20151016_009682	{memo X}
 QUEST_LV_0100_20151016_009683	$I will wait for Fabijus here.{nl}It'd be great to bring back a certain answer.
 QUEST_LV_0100_20151016_009684	$I am indeed Bartet.{nl}What's the matter?{nl}
 QUEST_LV_0100_20151016_009685	{memo X}
-QUEST_LV_0100_20151016_009686	
-QUEST_LV_0100_20151016_009687	
-QUEST_LV_0100_20151016_009688	
-QUEST_LV_0100_20151016_009689	
+QUEST_LV_0100_20151016_009686	She was strange.{nl}Since only the pilgrims visit this place, it was quite a strange sight.{nl}
+QUEST_LV_0100_20151016_009687	She was staring absently at something leather-like then suddenly teared it apart.{nl}To do such a feat with bare hands.{nl}
+QUEST_LV_0100_20151016_009688	It wasn't a strength of a human. I was trembling.{nl}She disappeared down there soon after throwing away the shredded leather.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9688]: the strength

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**